src/Components/ADT3DSceneBuilder/Internal/Shared/MakePivotRequired.tsx

```diff
@@ -1,21 +1,25 @@
 import { IPivotItemProps } from '@fluentui/react';
 import React from 'react';
+import { useTranslation } from 'react-i18next';
 import { customPivotItemStyles } from '../Behaviors/BehaviorsForm.styles';
 
 export function setPivotToRequired(
     isValid: boolean | undefined,
     link?: IPivotItemProps,
     defaultRenderer?: (link?: IPivotItemProps) => JSX.Element | null
 ): JSX.Element | null {
+    const { t } = useTranslation();
     if (!link || !defaultRenderer) {
         return null;
     }
     return (
         <span className={customPivotItemStyles.root}>
             {defaultRenderer({ ...link, itemIcon: undefined })}
-            {/* TODO: Add an aria label of some kind here for screen readers to see this error state */}
             {isValid === false && (
-                <span className={customPivotItemStyles.alert} />
+                <span
+                    aria-label={t('meshIsRequired')}
```

Comment:
This actually isnt a correct aria label since this can be used in contexts other than the elements tab.  I think a more appropriate label is probably "This field is required" or something, but if we feel that people should be able to customize it, they should use a prop to pass in the aria label